var/spack/repos/builtin/packages/hip/package.py

```diff
@@ -291,7 +291,10 @@ def get_paths(self):
             # We assume self.spec.prefix is /opt/rocm-x.y.z/hip and rocm has a
             # default installation with everything installed under
             # /opt/rocm-x.y.z
```

Comment:
I didn't update this comment in my inital patch - perhaps:
```suggestion
            # We assume self.spec.prefix is  /opt/rocm-x.y.z for rocm-5.2.0 and newer
            # and /opt/rocm-x.y.z/hip for older versions
```